tests/test_session_builder.c

```diff
@@ -135,13 +138,56 @@ START_TEST(test_schnorr_verification)
 
     signal_buffer_free(bob_signed_pre_key_public_serialized);
 
-    /* Have Alice process Bob's pre key bundle */
+    /* 
+        Alice processes Bob's pre key bundle.
+        She can verify Bob's Schnorr proof from within
+        session_builder_process_pre_key_bundle().
+    */
     result = session_builder_process_pre_key_bundle(alice_session_builder, bob_pre_key);
     ck_assert_int_eq(result, 0);
+     
+    /* Bob loads session state, including Alice's Schnorr proof */
+    result = signal_protocol_session_load_session(session_builder_get_store(alice_session_builder), &bob_record, session_builder_get_remote_address(alice_session_builder));
+    state = session_record_get_state(bob_record);
+
+    printf("loaded s_buf.....\n");
+    print(session_state_get_alice_s(state), session_state_get_alice_s(state)->len);
+    printf("loaded c_buf.....\n");
+    print(session_state_get_alice_c(state), session_state_get_alice_c(state)->len);
+    printf("loaded Xfull_buf.....\n");
+    print(session_state_get_alice_Xfull(state), session_state_get_alice_Xfull(state)->len);
+    printf("loaded Rfull_buf.....\n");
+    print(session_state_get_alice_Rfull(state), session_state_get_alice_Rfull(state)->len);
 
     /* 
-        Bob will verify Alice's Schnorr proof here...
+        Bob can verify Alice's Schnorr proof here... 
+        Access Alice's "s" by *state->alice_s_buf->data
```

Comment:
I wonder if it would be better if lines 166-190 were placed under one same function so that we could repeatedly use this for the other unit tests